YIF_XUnitTests/Integration/YIF_Backend/Controllers/InstitutionOfEducationAdminControllerTests.cs

```diff
@@ -248,5 +251,31 @@ public async Task AddInstitutionOfEducationModerator_Output_ByAddingSameModerato
a5ZRCSjH9Y1SRXM7Q8jAaZitl2dq7Lip1BEK8BQNZkFKlhsjoRv9Y6tEsnCp6ApFTUgnNtQMuojuWmilECqzp7bo9AB1AqN+hCvhCGcTFkJOEseXHdHtmmE1euBWTaRYhD5HdrRhWuojiRLwKY54VDJw+EwCycSpxUHL7tSOcdotTuCMjkaRWsEAPvPH2isulzUPzD0GVKwwCP2yX9kekSFsxIc0jyCx0eKtyHfCC9WTpqwG4ZZxz+1pNQgDoH86wWu6JmIKTLLu9S76/oxx/R21Kc92a/dV8opylomkCJmJbJzvJ947RNRu9RBaOzFq/7Sj0WP9MWp7I2w5SFeZHumFyaK4oEl2uWksxHJz5wdLtwcHEnSpb1LPBVk7I25IV/V/FqVp4cR+UWSOwdvdJCUIAYv3iSXBdxSnrC9r+h6y++pGFSkpCRXZIHI+IatCX9lXw9Y2iOydqUSpfduS5OMmprVkwQjsbNOa5Y/xf7YanGssOMukYM2eZuT0p8I8RZdod4CUgGiSc9C+dK9W0ePoaOxatZiB+ExcjsUNZvkj/8AUHsgux8ZPo+Rrs0yuyryLxdZ0ES1qJWlYwhKWNQTV6V+HWPrSOxcrWarokR2eyFnHimqHMoHuIXtj9Dg/h8fUuYK7t2fpWCLMiZVRxElufWPraOy1lGcyYeRSfZJi9HZmyfZmnmFj/QIXsiHrkz5hIUQgAjTIj3EKrwURhKSQxcH48Kx9qT2csg/uJiuqvioQB2h7IyJ0gplWdUuYFJwrxJDB6uSonDhJpvA3Q5eZSVUC8bR8o/bZyC7EKIAJzdhq7xxLn4ihCyQmgcklhQU3Uj7wLBYZYqiQlqbXd/GLE2uwoyXZk8u7+EZe34i+H5PgWBRomWpVWcIVXTSLJQMtQStCgo4mBdNGYHIuHxbuYj7wvtNYkf3yPwpWf8AKkxku2M67LZhUu1TJcxIAxolzFApBJZScFak1DGGpyfQlFLNnzJc6WHBI1yBr+UVTLal2QlXm/wg+wWeSibMQtYUgHZmKQsJUkEEHAUYgaVTkXIqKx3JuazFHeG3olFRJEvuZq1JTiLBRTQHCxbQlno8bexpIhwUm2CSL4VLCsKpgcuwSGJO8k08oir+mO4URvZCHNQc2dnGWtRkSIumXRJYlNtlqADh5akE1YgBRdxQh86tlANpsUtIJTPlrUzhIRMBPB8JS/M+UL2Ni4JMb2btKgN3ktC/4TgU33SAUpPNMaIzLNMlpnSFu5KVImOJiCa7QCsK0kk1AakZKxSJCpDU70vsvtlQdgAdCSMqMA7Vh/cvZ+XLlLXNmrTMSoOEJJGEOGTQiYCqhIbC2TFzEpVsqMfgUEWUktLmgFeI4phauejt1J4w5nKu9CCQiUpg4GKY6iBk7cMzvypGVskqaUhRlgJIKgStFUhyWBU5IYuGekDTpoIoQ/MfOM+CnpvHxhlbPpqOzVhLEykYjkMaiHbdiY+cVzezt3pOIy8BJLOqYKsfvDo0ZQWnu2KrXLUG8IDFJITUgeJmYgdHgG3X4oJSEzJbsWKSoEHi5fcNco85ePzN4kzaom0n3Pd8tKldwMlMSqYXLHMKVy0OcYKdaEbS02eWAGqVrLBTEbOJnYimkLJdqmzJie8KiFKFTxYO7coaS7tKUlIHebIDJEyp30duUdfia8cq8km29Gc4tq4nH7c1DZ5Xr84kF/sMzQn/AOGcfVqxI7Ofh+P/AKZcfIaZEqWMpeD/ANaly/8AIoD0ghMxT7M+ange7WnriRi/mgbrHmPjC42aWGJtdpBcTJC9wVLmSz/iStXtBMu9Zw8dnCv/AFTUK9Jgln3hYJgjsTucTwQ+THAvhAIxy50vnKmFPVSApPrBFnvGQskImSyrUYkv1DuIRItZGRI/XCLF2sLDLCVjcsBQ/mETwHzNIC7EV9o6Ct/vGSTJkjwyky+Mta5R/wDrIgiXbMOU+YOCiiYPNSMZ/wAUS/GPmjT4x7b46K/0Yzqbym/VXJmcClcs+YUqO03xM+tKrr3a0LH8ykk+ULgx8kaAEx7iaEKr3QA6lKT/ABy5ifVsJ6Ex1KvmWrKYg8lg+ghcGPkh5iHH3jxQAqP1lCWZfP3gOgPvFf77RrMD8KfBhD4MOSNCEmOwD+s4zv7/AEj69OYPwipfaYDwrfmkfKDhIOaH06xSlEqXKlk6koSSeZZzFZuqzf8AYldEgfKM4vtbMGQSejfGOFdsphp3cvqFfOH65EucTSfuay5qs6A54/Ax0m4LG3/Tp9fnSMirtZNr9HK5YVf7o5Pa+0NQIHEA/ExXrn9FzibI9nLER/YJHVX+6KJt1XcjxSZfHxH/AJjE2i/7RMzmHkKe0ATLRMP1g/Eq93il45dslzj0jczLLdKamXLPDCunJhAi7Rc6XPdI4ApV8Ywy+8OYlnniPvFeCY+Usckk+yhFev8ALJ5msmXjdiVhUtBSRTZQggilGcPzodS5rA0y/wCS5whRBdgRVzvzBHBgIzi5UzSZTikk+pjzupmsw9EIz6gw+CDkxpab2lqlqlBCkoXVSUqVgKqVIYPkPIZQqtE5BQlKZWAj6yQra5pJIf8AhblEMg6zVHokeyY8NkGZXMP4lCvRoOC+ByYsXZmOJSstCkuWbiQICvFLBJ3vpyh6qwpLOVH8SvnFC7ql6gn8SvnC4/gVmfRalpYhZpUAmgIypBSe0c4fXHkmDZl0S9AfMwuFyFLmh3PRoUvGnlqylIv/AKUT/tj+WJA37oX9kef5R5C4L4PkfSL4DKU1OVITTlneY9iRtHZmypM1X2j5mL0zVfaPmY9iRoSzxSzvMVYzvMSJAJFyFneY9VEiRHY3opMeGPIkaISJJmq+0fMx2EhQ2hi51949iRLGL5qyMiRXSkMJWUSJEIo7EdCJEgDo8MV6xIkAiwRzEiQ30B1E3RIkMDpXxisxIkAiJzjlUSJCF2RX6844/P2iRICjxGUSZ84kSEB4qKlxIkJASJEiRQH/2Q==" }),
```

Comment:
there is sooo long line, so I think you should place image path to separated region for example, just to make code more readable